### PR TITLE
Task: update ember-submission-form-fields version

### PR DESCRIPTION
**Pull request by wolfderechter**
## ID
DGS-184

## Description

Bumped the ember-submission-form-fields to latest version because we need it for a new subsidy.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## Links to other PR's

- https://github.com/lblod/app-digitaal-loket/pull/554

### CODE REVIEW

File: package-lock.json
Comment (Windvis) [inline package-lock.json:1]: Did you rerol the lock file by any chance? It seems like an awful lot of changes for a simple version bump of an already installed package. What version of npm are you using?

Comment (Windvis) [reply package-lock.json:1]: Something strange is still going on. You don't have any changes when you `npm install` locally?

Comment (Windvis) [reply package-lock.json:1]: > Ahh my base branch was a few commits behind, fixed

Yea, I would try rebasing, to be sure.

Comment (wolfderechter) [reply package-lock.json:1]: Removed my node_modules and npm installed again, should be good now

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "version": "0.91.1",
       "license": "MIT",
       "devDependencies": {
-        "@appuniversum/ember-appuniversum": "^3.4.1",
+        "@appuniversum/ember-appuniversum": "^3.3.0",
```
Comment (Windvis) [inline package-lock.json:12]: It's installing an older version of Appuniversum again 😄.

**Pull request closed without merging**